iter_app/ObjectRepresentation[] objects
---
bool status
